hexrd/config/root.py

```diff
@@ -16,8 +17,42 @@
 
 class RootConfig(Config):
 
+    @property
+    def working_dir(self):
+        """Working directory, either specified in file or current directory
+
+        This directory is certain to exist. If the specified directory does
+        not exist, it defaults to the current working directory.
+        """
+        try:
+            temp = Path(self.get('working_dir'))
+            if not temp.exists():
+                raise IOError(f'"working_dir": {temp} does not exist')
+            return temp
+
+        except RuntimeError:
```

Comment:
What causes the `RuntimeError` to occur? Looks like the above `IOError` is not a `RuntimeError`.

Oh, maybe it occurs if `self.get('working_dir')` returns `None`? If that's the case, it might be clearer to just check for that in an `if/else` statement rather than catch a `RuntimeError`.

Reply:
Thanks Don! That makes it easier to read 🙂 . It just wasn't clear before what was causing the `RuntimeError` that was being caught.